storage/src/mmr/mod.rs

```diff
@@ -76,6 +76,7 @@ mod iterator;
 pub mod journaled;
 pub mod mem;
 pub mod mutable;
+pub mod operation_array;
```

Comment:
nit: suggest just calling this `operation`

Reply:
done